libp2p/protocols/pubsub/rpc/protobuf.nim

```diff
@@ -132,8 +132,8 @@ proc decodeGraft*(pb: ProtoBuffer): ProtoResult[ControlGraft] {.
 
   trace "decodeGraft: decoding message"
   var control = ControlGraft()
-  if ? pb.getField(1, control.topicId):
-    trace "decodeGraft: read topicId", topic_id = control.topicId
+  if ? pb.getField(1, control.topicID):
```

Comment:
Erm, we switched to PeerId as part of the first styleCheck cleanup, so we should try to stay consistent with the `Id`, if possible (also applies to `MessageID`)

Not sure why theses were not catch on the first pass, sorry about that

Reply:
https://github.com/status-im/nim-libp2p/pull/743/commits/90de6d56657ee6d3fd86faa2f6ed14aae06ec0ab

Got those two in particular. There are some similar but not-exactly-identical identifiers (plural versions, for example) which I didn't change, because better not to do more than people want then revert.

Reply:
I mean, it's now or never to fix these, so feel free to fix any inconsistencies you find

Reply:
Okay, with this kind of diffuse change, probably worth merging this PR and doing the rest in another PR.